clusters/overlays/volante/kustomization.yaml

```diff
@@ -3,4 +3,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 bases:
-  - ../../../components/apps/sriov-operator
+  - ../../../components/apps/sriov-operator/overlays/default
```

Comment:
Should we align the naming of the application with the name of the Operator?

SR-IOV = openshift-sriov-network-operator
PTP = openshift-ptp

Reply:
Agree. Done.